src/main/java/com/slack/kaldb/server/KaldbLocalSearcher.java

```diff
@@ -39,28 +39,32 @@ public static SearchQuery fromSearchRequest(KaldbSearch.SearchRequest searchRequ
   }
 
   public static SearchResult<LogMessage> fromSearchResultProto(
-      KaldbSearch.SearchResult protoSearchResult) throws IOException {
-    List<LogMessage> hits = new ArrayList<>(protoSearchResult.getHitsCount());
+      KaldbSearch.SearchResult protoSearchResult) {
```

Comment:
I think it may be better to keep the original fromSearchResultProto method as is and add a wrapper method that performs this operation. That way we can save a refactoring if we need the underlying method elsewhere.

Reply:
I'm unsure what you are suggesting - all the change does is instead of throwing the IOException it returns empty.

Are you saying that we should move the returning default to another method?